clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/cache/DefaultCacheDataBuilder.groovy

```diff
@@ -28,7 +30,7 @@ class DefaultCacheDataBuilder {
     new DefaultCacheData(id, ttlSeconds, attributes, relationships)
   }
 
-  public static Map<String, DefaultCacheDataBuilder> defaultCacheDataBuilderMap() {
-    return [:].withDefault { String id -> new DefaultCacheDataBuilder(id: id) }
+  public static ConcurrentHashMap<String, DefaultCacheDataBuilder> defaultCacheDataBuilderMap() {
```

Comment:
Last tiny thing: I'd suggest making this a `ConcurrentMap` in the interface.  The consumer cares that it's concurrent and that it's a map, but the fact that it's a hash map is an implementation detail.